ui/components/members/BulkAllocateModal.js

```diff
@@ -0,0 +1,92 @@
+import { useState } from "react";
```

Comment:
Could maybe be nice to refactor this file and AllocateModal to deduplicate code in some way. But it's not a ton of duplication so only if you feel like it

Reply:
Let's go for rule of three, "Three strikes and you refactor" :-)